EIPS/eip-2537.md

```diff
@@ -17,17 +17,17 @@ This precompile adds operation on BLS12-381 curve as a precompile in a set neces
-- BLS12_G2ADD - to perform point addition on a curve twist defined over quadratic extension of the base field
-- BLS12_G2MUL - to perform point multiplication on a curve twist defined over quadratic extension of the base field
-- BLS12_G2MULTIEXP - to perform multiexponentiation on a curve twist defined over quadratic extension of the base field
-- BLS12_PAIRING - to perform a pairing operations between a set of *pairs* of (G1, G2) points
-- BLS12_MAP_FP_TO_G1 - maps base field element into the G1 point
-- BLS12_MAP_FP2_TO_G2 - maps extension field element into the G2 point
+- BLS12_G1ADD - to perform point addition in G1 (curve over base prime field) with a gas cost of `500` gas
+- BLS12_G1MUL - to perform point multiplication in G1 (curve over base prime field) with a gas cost of `12000` gas
+- BLS12_G1MULTIEXP - to perform multiexponentiation in G1 (curve over base prime field) with a gas cost formula defined in the corresponding section
+- BLS12_G2ADD - to perform point addition in G2 (curve over quadratic extension of the base prime field) with a gas cost of `800` gas
+- BLS12_G2MUL - to perform point multiplication in G2 (curve over quadratic extension of the base prime field) with a gas cost of `45000` gas
+- BLS12_G2MULTIEXP - to perform multiexponentiation in G2 (curve over quadratic extension of the base prime field) with a gas cost formula defined in the corresponding section
+- BLS12_PAIRING - to perform a pairing operations between a set of *pairs* of (G1, G2) points a gas cost formula defined in the corresponding section
+- BLS12_MAP_FP_TO_G1 - maps base field element into the G1 point with a gast cost of `5500` gas
+- BLS12_MAP_FP2_TO_G2 - maps extension field element into the G2 point with a gas cost of `75000` gas
 
-Mapping functions are implemented according to [IEFT specification](https://github.com/cfrg/draft-irtf-cfrg-hash-to-curve/blob/master/draft-irtf-cfrg-hash-to-curve.md#deterministic-mappings-mappings) version `v7`(!) using an simplified SWU method. It does NOT perform mapping of the byte string into field element that can be implemented in many different ways and can be efficiently performed in EVM, but only does field arithmetic to map field element into curve point. Such functionality is required for signature schemes.
+Mapping functions are implemented according to [IEFT specification](https://github.com/cfrg/draft-irtf-cfrg-hash-to-curve/blob/master/draft-irtf-cfrg-hash-to-curve.md#deterministic-mappings-mappings) version `v7` using an simplified SWU method. Latter methods do not change a part that the corresponding precompiles implement. Mapping function does NOT perform mapping of the byte string into field element (as it can be implemented in many different ways and can be efficiently performed in EVM), but only does field arithmetic to map field element into curve point. Such functionality is required for signature schemes.
```

Comment:
We shouldn't be referring to draft specifications at all, and definitely not draft specifications in external repositories.  We have made a few exceptions on external links when linking to other standards bodies like RFC and IETF, and I can imagine adding IRTF to the list but definitely not to drafts.

If this is critical to the specification then you'll need to add it to the Specification section as either inline content or an asset in `../assets/eip-2537/*`.  You could make a copy of the draft specification and put it in assets if you wanted, but be aware that this spec will be bound to the version of the draft that is in the assets folder, not the version that IRTF ends up publishing so you should **NOT** call it an IEFT specification nor name it such.  We don't want another keccak256/sha3 situation on our hands... cleaning up that mess took *years* (and we still deal with it today).